src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -249,8 +249,6 @@ public void syncPostBatch(final String topicId, final List<BatchItem> batch) thr
                 if (circuitBreaker.allowRequest()) {
                     sendFutures.put(item, publishItem(producer, topicId, item, circuitBreaker));
                 } else {
-                    LOG.warn("Short circuiting request to Kafka due to timeout for topic {} on broker with id {}. {}",
-                            topicId, item.getBrokerId(), circuitBreaker.getMetrics());
```

Comment:
We need to log at least something - maybe it will be good to log at the end of request how many events were shurt-cirquited